Documentation/async-disaster-recovery.md

```diff
@@ -0,0 +1,454 @@
+* As you are creating the static binding between PVC and PV, a new PV won’t
+ be created here, the PVC will get bind to the existing PV.
+* Create the VolumeReplicationClass on the secondary site.
+* Create the VolumeReplications for all the PVC’s for which mirroring
+ is enabled
+  * `replicationState` should be `primary` for all the PVC’s on
+   the secondary site.
+* Check whether the image is marked `primary` on the secondary site
+ by verifying it in VolumeReplication CR status.
+* Once the Image is marked as `primary`, the PVC is now ready
+ to be used. Now, we can scale up the applications to use the PVC.
+
+>:memo: **WARNING**: In Async Disaster recovery use case, we don't get
+> the complete data.
+> We will only get the crash-consistent data based on the snapshot interval time.
+
+### Failback
+
+To perform a failback operation to primary cluster in case of planned migration
+, just repeat the Failback steps in vice-versa.
+
+>:memo: **Remember**: We can skip the backup-restore operations
+> in case of failback if the required yamls are already present on
+> the primary cluster. Any new PVCs will still need to be restored on the
+> primary site.
+
+## Disaster Recovery
+
+> Use cases: Natural disasters, Power failures, System failures, and crashes, etc.
+
+### Failover (abrupt shutdown)
+
+In case of Disaster recovery, create VolumeReplication CR at the Secondary Site.
+ Since the connection to the Primary Site is lost, the operator automatically
+  sends a GRPC request down to the driver to forcefully mark the dataSource as `primary`.
+
+* If you are manually creating the PVC and PV on the secondary cluster, remove
+ the claimRef section in the PV objects.
+* Create the storageclass, PVC, and PV objects on the secondary site.
+* As you are creating the static binding between PVC and PV, a new PV won’t be
+ created here, the PVC will get bind to the existing PV.
+* Create the VolumeReplicationClass and VolumeReplication CR on the secondary site.
+* Check whether the image is `primary` on secondary site, by verifying in
+ the VolumeReplication CR status.
+* Once the Image is marked as `primary`, the PVC is now ready to be used. Now,
+ we can scale up the applications to use the PVC.
+
+### Failback (post-disaster recovery)
+
+Once the failed cluster is recovered on the primary site and you want to failback
+ from secondary site, follow the below steps:
+
+* Update the VolumeReplication CR replicationState
+ from `primary` to `secondary` on the primary site.
+* Scale down the applications on the secondary site.
+* Update the VolumeReplication CR replicationState state from `primary` to
+ `secondary` in secondary site.
+* On the primary site, verify that the VolumeReplication status is marked as
+ volume ready to use
+* Once the volume is marked to ready to use, change the replicationState state
+ from `secondary` to `primary` in primary site.
+* Scale up the applications again on the primary site.
```

Comment:
This step could be confusing as user never scaled down the application in the `primary` site which was down due to a disaster.

Reply:
Good catch! Added a step to scale down the applications on the primary cluster, and make sure that all the PVCs in workload are no longer in use.